resources/patches/ruby_27/rexml_for_CVE-2024-35176.patch

```diff
@@ -0,0 +1,135 @@
+commit e597f07718c27dc4414ad39f121376e53056475a
+Author: Shubham Shinde <shubham.shinde@perforce.com>
+Date:   Tue Jul 9 19:40:43 2024 +0530
+
+    Read quoted attributes in chunks (#126)
+    
+    
+    # Conflicts:
+    #       lib/rexml/parsers/baseparser.rb
+    #       lib/rexml/source.rb
```

Comment:
I wouldn't expect conflicts. Did this occur because you cherry-picked the commit into the ruby 2.7 source tree? Is there a different patch we can use that applies cleanly?

Reply:
Hi @joshcooper,
Yes, I cherry-picked the commit https://github.com/ruby/rexml/commit/4325835f92f3f142ebd91a3fdba4e1f1ab7f1cfb into tag https://github.com/ruby/ruby/releases/tag/v2_7_8 and it conflicted. 
Was not able to find other patches to apply cleanly either.

Reply:
We had to resolve conflicts manually, is there any other path we can take in such situations?